frontend/view-edit-client/client-home/view-edit-intake-info.component.tsx

```diff
@@ -143,15 +149,61 @@ export default function ViewEditIntakeInfo(props: ViewEditIntakeInfoProps) {
       dateOfIntake: intakeDateInputRef.current.value,
       couldVolunteer: clientSourceRef.current.couldVolunteer,
       clientSource: clientSourceRef.current.clientSource,
-      intakeServices: intakeServicesRef.current.checkedServices
+      intakeServices: intakeServicesRef.current.checkedServices.map(
+        service => service.id
+      )
     };
     dispatchApiStatus({
       type: "do-patch",
       newClientData
     });
   }
+
+  function couldVolunteer() {
```

Comment:
I found a bug where we weren't showing whether a client could volunteer on the View Client page.